x-pack/plugins/siem/server/lib/detection_engine/routes/index/ecs_mapping.json

```diff
@@ -441,6 +444,141 @@
           }
         }
       },
+      "dll": {
+        "properties": {
+          "code_signature": {
```

Comment:
@marshallmain Are we going to address splitting these (trusted/untrusted) in a later PR?

Reply:
I'm working on a plan to address it with the ECS team. I think multiple code signatures is something we should address at the ECS level rather than making our own custom changes to it.

Reply:
Thinking about it more, addressing at the ECS level may take some time. I'd like to merge this so we can start and then merge more changes in future PRs.

Reply:
Sounds good, thanks!